### PR TITLE
post-merge updates: security page

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

Updates security page ready for post-merge. Moves "Eth2" scam to more prominent position on page.

## Related Issue

#7075

### CODE REVIEW

File: src/content/security/index.md [hunk 1]
```diff
@@ -240,7 +248,7 @@ These fraudulent brokers find their targets by using fake accounts on YouTube to
...
+Ethereum mining was switched off in 2022. However, mining pool scams still exist. Mining pool scams involve people contacting you unsolicited, and claiming that you can make large returns by joining an Ethereum mining pool. The scammer will make claims and stay in contact with you for however long it takes. Essentially, the scammer will try and convince you that when you join an Ethereum mining pool, your cryptocurrency will be used to create ETH and that you will be paid dividends in the form of ETH. What will end up happening is, you will notice that your cryptocurrency is making small returns. This is simply to bait you into investing more. Eventually, all of your funds will be sent to an unknown address and the scammer will either disappear or in some cases will continue to stay in touch as has happened in a recent case.
```
Comment (minimalsm) [inline src/content/security/index.md:251]: Nitpick: switched off might be unclear. Don't think it's important if you disagree.

```suggestion
As of September 2022, mining on Ethereum is no longer possible. However, mining pool scams still exist. Mining pool scams involve people contacting you unsolicited and claiming that you can make large returns by joining an Ethereum mining pool. The scammer will make claims and stay in contact with you for however long it takes. Essentially, the scammer will try and convince you that when you join an Ethereum mining pool, your cryptocurrency will be used to create ETH and that you will be paid dividends in the form of ETH. What will end up happening is, you will notice that your cryptocurrency is making small returns. This is simply to bait you into investing more. Eventually, all of your funds will be sent to an unknown address, and the scammer will either disappear or in some cases will continue to stay in touch as has happened in a recent case.
```

File: src/content/security/index.md [hunk 2]
```diff
@@ -214,6 +214,14 @@ As a general rule, staff will never communicate with you through private, unoffi
   </div>
 </InfoBanner>
 
+### 'Eth2' token scam {#eth2-token-scam}
+
+In the run up to [The Merge](/upgrades/merge/), scammers took advantage of the confusion around the term 'Eth2' to try and get users to redeem their ETH for an 'ETH2' token. There is no 'ETH2' and no other legitimate token was introduced with The Merge. The ETH that you owned before The Merge is the same ETH now, there is **no need to take any action related to your ETH to account for the switch from proof-of-work to proof-of-stake**.
```
Comment (minimalsm) [inline src/content/security/index.md:219]:
```suggestion
In the run-up to [The Merge](/upgrades/merge/), scammers took advantage of the confusion around the term 'Eth2' to try and get users to redeem their ETH for an 'ETH2' token. There is no 'ETH2', and no other legitimate token was introduced with The Merge. The ETH that you owned before The Merge is the same ETH now. There is **no need to take any action related to your ETH to account for the switch from proof-of-work to proof-of-stake**.
```

File: src/content/security/index.md [hunk 3]
```diff
@@ -214,6 +214,14 @@ As a general rule, staff will never communicate with you through private, unoffi
...
+
+Scammers may appear in the form of "support" telling you if you deposit your ETH you will receive back 'ETH2'. There is no [official Ethereum support](/community/support/), and there is no new token. Never share your wallet seed phrase with anyone.
```
Comment (minimalsm) [inline src/content/security/index.md:221]:
```suggestion
Scammers may appear as "support", telling you that if you deposit your ETH, you will receive back 'ETH2'. There is no [official Ethereum support](/community/support/), and there is no new token. Never share your wallet seed phrase with anyone.
```

**Review by minimalsm — APPROVED** [review]
Suggested some small revisions but LGTM

**Review by GustavoRSSilva — APPROVED** [review]

**Review by minimalsm — APPROVED** [review]

**Pull request merged**